compiler/optimizer/OSRDefAnalysis.cpp

```diff
@@ -910,6 +912,11 @@ int32_t TR_OSRLiveRangeAnalysis::perform()
    return 0;
    }
 
+// 
+// To reduce the impact of pending push temps in postExecutionOSR, the live pending push symrefs can be stored or
+// referenced in anchored loads, between the OSR point and the transition. In the event that the anchored loads
+// are not commoned, they will be removed.
+//
```

Comment:
If the load was anchored purely to inform this stage of liveness, it would exist with a reference count of 1. If it was instead anchored to avoid side effects of a pending push store, a reference count greater than 1 would be expected, and therefore can't be removed.